tests/__tests__/auto-cleanup.js

```diff
@@ -0,0 +1,19 @@
+import { render } from '@testing-library/vue'
+import '@testing-library/jest-dom/extend-expect'
+
+const HelloWorldComponent = {
+  template: `<h1>Hello World</h1>`
+}
+// This just verifies that by importing VTL in an
+// environment which supports afterEach (like jest)
+// we'll get automatic cleanup between tests.
+test('render the first component', () => {
+  const { container } = render(HelloWorldComponent)
```

Comment:
I like how you inlined the component in the `auto-cleanup-skip.js` test. Could we do the same thing here, instead of creating a "global" `HelloWorldComponent`?

Reply:
Done!